common/customizable_localization/wc_magical_lifestyles_custom_loc.txt

```diff
@@ -386,4 +386,345 @@ GetWCMagicalLifestyleElementalEarthMagicNextTraitForPerks = {
 
 		localization_key = "wc_elemental_earth_magic_next_trait_level_is_1"
 	}
+}
+
+GetWCMagicalLifestyleLightHowMuchPerksForNextTrait = {
```

Comment:
Keep only one way. Both numbers tell the same info.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/132129128-d08fd191-e1fa-4c76-adb3-5f41093ae9ef.png)

</details>

Reply:
Well, no countdown then..